### ISSUE TITLE
'invalid format' status when inputting IPv6 addresses

### ISSUE THREAD

**Issue by codersaur** [bartbutenaers/node-red-contrib-ip-location-lite#1]
Although the readme claims this supports IPv6, the input filter is rejecting anything except dotted quad (IPv4) notation.
Also, it's not throwing an error, it's just updating the node.status to 'invalid format'.

Suggest it needs updating to allow IPv6 address formats. The underlying geoip-lite package seems to support full and condensed IPv6 notation. Thanks.

**Comment by bartbutenaers** [comment]
Hi @codersaur,

I have update the regex (in this Github repo) that validates the ip address.  
Would be nice if you could test it, because there are lots of different regexes available for this purpose.  Did a quick test, but I'm not an ipv6 expert...

 You can install the version from Github by executing following command from within your .node-red directory:
```
npm install bartbutenaers/node-red-contrib-ip-location-lite
```

Note that I need to have surgery this friday, and I won't be able to develop for a couple of months.  So not sure whether I will get it published on NPM tomorrow...

Bart

**Comment by bartbutenaers** [comment]
Hi @codersaur,

At the end I have removed my regex check, since the geoip-lite package has its own checks.  That way I don't add any extra restrictions, so you should be able to inject all the input that geop-lite accepts.

This is now available as version ***1.0.1*** in the palette:

![image](https://user-images.githubusercontent.com/14224149/122639775-0fc77d00-d0fc-11eb-9eaa-d9a34ac5878e.png)

Feel free to reopen this issue, if it doesn't fit your needs ...
Bart
